README.md

```diff
@@ -1,2 +1,125 @@
-# retorch-llm-rp
-Replication package for LLM experimentation
+# Replication package for *'Exploratory study of the usefulness of LLMs in System testing'*
+
+This repository contains the replication package of the paper *Software System Testing assisted by Large Language
+Models: An Exploratory Study*
+published at [TO-DO]
+
+The replication package comprises the test scripts used to generate the test scenarios and system test code as well
+as the different inputs required: the user requirements, the system test cases provided as example, the test scenarios
+used as input and the scenario used as example. The replication package also provides the different outputs of our
+exploratory study in the `\docs` folder, the original raw data is available in the [ZENODO](TO-DO) repository.
```

Comment:
añadir link a zenodo ahora que está

Reply:
I am waiting for @ClaudiodelaRiva  approval of the Zenodo draft, at the moment it would be available I update everything